scripts/pugbomb.py

```diff
@@ -5,9 +5,15 @@
 
 def pugbomb(response, args):
     #gets the number
-    num = int(args[0])
+    try:
+        num = round(int(args[0]))
+    except:
+        return "`Please enter a number next time`"
```

Comment:
I might be missing something, but is there a reason you nead `"` and `'` in the same string?